frontend/src/components/pages/NotFound.tsx

```diff
@@ -0,0 +1,39 @@
+import { ArrowLeftIcon } from '@chakra-ui/icons';
+import { Box, Button, Center, Heading, Text } from '@chakra-ui/react';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const NotFound: React.FC = () => (
+  <Center>
+    <Box
+      h="container.sm"
+      w="container.md"
+      mt="32"
+      display="flex"
+      flexDirection="column"
+      alignContent="center"
+      justifyContent="space-between"
+      bg="red.200"
+      borderRadius="3xl"
+      backgroundImage="url('https://media.istockphoto.com/vectors/error-page-or-file-not-found-icon-vector-id924949200?b=1&k=6&m=924949200&s=612x612&w=0&h=eonpW4ymWtCIKLt8k85EhAZ32kiJ-v_LmTEkUcPmg7Q=')"
+      backgroundSize="cover"
+      backgroundPosition="center"
+      backgroundRepeat="no-repeat"
```

Comment:
Should either be exported to theme / config or should be put in a css so it is re-usable